components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -108,7 +110,7 @@ class NewTabPage extends React.Component<Props, State> {
 
   componentDidMount () {
     // if a notification is open at component mounting time, close it
-    this.props.actions.showGridSiteRemovedNotification(false)
+    this.props.actions.showTilesRemovedNotice(false)
```

Comment:
I know this isn't a new addition, just a rename, but there's no need for a redux action call and a whole re-render cycle. This should be simply handled in the initial init redux reducer action.

Reply:
Because all new tabs are sharing the same state, I think the intention was if you have the notice on one tab- then it would hide when opening a new tab. When the state is initialized, it's set to `false`. I can definitely remove (since the notice does auto-hide itself after 3 seconds anyways). LMK 😄

Reply:
Then it could be set on an existing init redux action and not cause a redux -> render -> redux -> render cycle. But it's only once so we can get to this change on the next refactor 👍